### ISSUE TITLE
Splunk Operator: upgrade instructions outdated (for 1.0.5 and older)

### ISSUE THREAD

**Issue by magguns** [splunk/splunk-operator#994]
### Please select the type of request

Enhancement

### Tell us more

The documentation describing the upgrade process for [1.0.5 and older](https://splunk.github.io/splunk-operator/SplunkOperatorUpgrade.html#steps-to-upgrade-from-105-or-older-version-to-latestl) seems to be outdated/broken.

**The first step describes downloading the upgrade script:**
```
wget -O operator-upgarde.sh https://github.com/splunk/splunk-operator/releases/download/2.1.0/operator-upgrade.sh
``` 
This `operator-upgrade.sh` is however missing from the 2.0.x releases and results in a 404. The last release to include an upgrade script is 1.1.0: https://github.com/splunk/splunk-operator/releases/download/1.1.0/upgrade-to-1.1.0.sh

**The second step describes downloading the install manifest:**
```
wget -O splunk-operator-install.yaml https://github.com/splunk/splunk-operator/releases/download/2.1.0/splunk-operator-install.yaml
```
This is also missing from 2.0.x releases and results in a 404. Again the last release to include a `splunk-operator-install.yaml` is 1.1.0: https://github.com/splunk/splunk-operator/releases/download/1.1.0/splunk-operator-install.yaml

Minor nitpicks are that the first and second steps are both labeled `1. ...`, aswell as a small typo in `wget -O operator-upgarde.sh ...`

I think I'll go from 1.0.5 to 1.1.0 before upgrading to 2.1.0 just to be safe here. Perhaps an easy fix to the docs would be to simply recommend an intermediate step to 1.1.0?

**Comment by gaurav-splunk** [comment]
Thanks @magguns for catching this. We will fix this in our docs.

**Comment by vivekr-splunk** [comment]
@magguns 2.1.1 patch release should have updated documentation.

**Comment by gaurav-splunk** [comment]
@magguns can we close this now since we have updated the documentation in 2.1.1 release?

**Comment by vivekr-splunk** [comment]
@magguns can we close this issue since its documented

**Comment by gaurav-splunk** [comment]
Closing it since it is already fixed in 2.1.1 release.
